server/src/infra/repositories/asset.repository.ts

```diff
@@ -523,11 +523,11 @@ export class AssetRepository implements IAssetRepository {
       builder = builder.leftJoin('asset.albums', 'album').andWhere('album.id = :albumId', { albumId });
     }
 
-    if (userId) {
-      builder = builder.andWhere('asset.ownerId = :userId', { userId });
+    if (userIds) {
+      builder = builder.andWhere('asset.ownerId IN (:...userIds )', { userIds });
```

Comment:
I wonder what the performance impact of this is.

Reply:
### Using equal
![equal](https://github.com/immich-app/immich/assets/27055614/452fbae2-4a46-43b8-b954-30f032afd615)

## Using In
![In](https://github.com/immich-app/immich/assets/27055614/03ac70f0-c239-4229-8565-2cf5a2716c83)

Not showing any clear differences